client/webpack.config.js

```diff
@@ -0,0 +1,94 @@
+const path = require('path');
```

Comment:
Any particular reason to go with a more custom webpack config over create-react-app?